.stickler.yml

```diff
@@ -0,0 +1,8 @@
+linters:
+  flake8:
+  eslint:
+    config: './eslint.config'
+  stylelint:
+files:
+  ignore:
+    - '.eslintrc.json'
```

Comment:
Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/react-redux).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

Reply:
Thank you for your observations. I've made the necessary changes.